### PR TITLE
1 create skeleton of project

### PR DESCRIPTION
**Pull request by PanLeon**
## Trello

* [Main Trelo ticket](https://trello.com/c/8u3pcJXX/1-create-skeleton-of-project)

## Code reviewers

- [x] @Ashiato
- [ ] @black-art
- [x] @hrytskivr

### Second Level Review

- [ ] @byshovets
- [ ] @lhalam

## Summary of issue

created skeleton of app

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -99,3 +99,5 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+.idea/
```
Comment (lhalam) [inline .gitignore]: add endline

File: myTrip/myTrip/settings.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
+"""
+Django settings for myTrip project.
+
+Generated by 'django-admin startproject' using Django 1.11.3.
```
Comment (lhalam) [inline myTrip/myTrip/settings.py]: delete this line

File: myTrip/myTrip/settings.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/1.11/howto/static-files/
+
+STATIC_URL = '/static/'
+STATICFILES_DIRS = [os.path.join(BASE_DIR, 'static'), ]
+
+try:
+    from .local_settings import *
+except:
+    pass
```
Comment (lhalam) [inline myTrip/myTrip/settings.py]: add endline

File: myTrip/myTrip/settings.py [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+]
+
+WSGI_APPLICATION = 'myTrip.wsgi.application'
+
+
+
+# Password validation
+# https://docs.djangoproject.com/en/1.11/ref/settings/#auth-password-validators
+
+AUTH_PASSWORD_VALIDATORS = [
```
Comment (lhalam) [inline myTrip/myTrip/settings.py]: delete this block

File: requirements.txt [hunk 5]
```diff
@@ -0,0 +1,3 @@
+Django==1.11.3
+psycopg2==2.7.1
+pytz==2017.2
```
Comment (hrytskivr) [inline requirements.txt:3]: delete this req

**Review by lhalam — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (hrytskivr) [reply .gitignore]: done

File: myTrip/myTrip/settings.py (hunk 3 above)
Comment (hrytskivr) [reply myTrip/myTrip/settings.py]: done

File: requirements.txt (hunk 5 above)
Comment (hrytskivr) [reply requirements.txt:3]: required dependency, cannot be deleted

**Review by hrytskivr — APPROVED** [review]
Nice1

**Review by lhalam — APPROVED** [review]
good

**Pull request merged**